embed.js

```diff
@@ -0,0 +1,67 @@
+var fs = require("fs");
+var http = require("http");
+var https = require("https");
+var url = require("url");
+var express = require('express');
+var cors = require('cors');
+var app = express();
+var crypto = require('crypto');
+const path = require('path');
+app.use(cors());
+// Parse JSON bodies (as sent by API clients)
+app.use(express.json());
+
+//assign a port number for an API to run,
```

Comment:
//Assign a port number for an API to run.

Reply:
updated this line